File: src/main/java/com/eric/okpo_java/lab1/Task5.java
package com.eric.okpo_java.lab1;

import java.time.Instant;
import java.util.Calendar;
import java.util.Date;
import java.util.Scanner;

public class Task5 {

    public static void main(String[] args) {
        int year, nowYear, age;
        var input = new Scanner(System.in);
        System.out.println("Введите год рождения пользователея");
        year = input.nextInt();
        nowYear = Calendar.getInstance().getWeekYear();
        age = nowYear - year;

        System.out.println(age);
    }
}

File: src/main/java/com/eric/okpo_java/lab4/Task5.kt
package com.eric.okpo_java.lab4

class Task5 {

    fun main() {
        var sum = 0
        var count = 0

        for (i in 0..100 step 7) {
            sum += i
            count++
        }

        println("Sum: $sum, Count: $count")
    }
}